weblate/utils/csv.py

```diff
@@ -5,4 +5,17 @@
 # The following characters are considered problematic for CSV files
 # - due to how they are interpreted by Excel
 # - due to the risk of CSV injection attacks
-PROHIBITED_INITIAL_CHARS = {"=", "+", "-", "@", "|", "%"}
+import sys
+import unicodedata
+
+csv_formula_triggers = {"=", "+", "-", "@", "|", "%"}
+unicode_whitespaces = {
+    chr(c) for c in range(sys.maxunicode + 1) if unicodedata.category(chr(c)) == "Zs"
+}
+
+PROHIBITED_INITIAL_CHARS = csv_formula_triggers | unicode_whitespaces
+
+# escape the whitespaces so they are rendered as their string representation instead of an actual character
+PROHIBITED_INITIAL_CHARS_FOR_DISPLAY = {"=", "+", "-", "@", "|", "%"} | {
```

Comment:
Avoid defining the same set again:

```suggestion
PROHIBITED_INITIAL_CHARS_FOR_DISPLAY = csv_formula_triggers | {
```